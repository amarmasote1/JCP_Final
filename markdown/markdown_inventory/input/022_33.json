
{
	"key": "JCP|1542|70630003035|EACH"
}  