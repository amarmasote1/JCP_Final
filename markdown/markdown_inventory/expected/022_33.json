{
    "warnings": null,
    "message": "Success",
    "body": {
        "dirty": false,
       
        "shipSafetyFactorSource": 0,
        "bopusSafetyFactor": 2,
        "shipSafetyFactor": 1,
        "onhandSupply": 1,
        "reservedDemand": 0,
        "pickSafetyFactorSource": 0,
        "unconfirmedDemand": 0
    },
    "key": "JCP|1542|70630003035|EACH",
    "status": 200
}